.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+# What does this change intend to accomplish?
+
+ 
+### All Submissions:
```

Comment:
I'm not really sure we need to break things up into All/New Features/Core Changes categories, seems like all of these checks apply to any changes. Also, tests passing is listed under both New Features/Core Changes.

Reply:
totally. it's pretty condensed in the latest rendition; check it out :+1: